### PR TITLE
LPS-94798 Remove the adjustment of align point

### PR DESCRIPTION
**Pull request by gregory-bretall**
Fixes both:
https://issues.liferay.com/browse/LPS-94798
https://issues.liferay.com/browse/LPS-94822

Test Results: https://github.com/gregory-bretall/liferay-portal/pull/193#issuecomment-488465713
https://github.com/gregory-bretall/liferay-portal/pull/193#issuecomment-488519174

@fortunatomaldonado

### CODE REVIEW

File: modules/apps/frontend-js/frontend-js-aui-web/src/main/resources/META-INF/resources/liferay/menu.js [hunk 1]
```diff
@@ -348,8 +348,6 @@ AUI.add(
 
 					var modalMask = false;
 
-					align.points = instance._getAlignPoints(cssClass);
-
 					menu.addClass('lfr-icon-menu-open');
```
Comment (blzaugg) [inline modules/apps/frontend-js/frontend-js-aui-web/src/main/resources/META-INF/resources/liferay/menu.js]: I think you can remove the rest of the `align` uses, as it's no longer being changed.

```diff
- var align = overlay.get('align');
...
- align: align,
```

Comment (blzaugg) [reply modules/apps/frontend-js/frontend-js-aui-web/src/main/resources/META-INF/resources/liferay/menu.js]: Please retest as well.

**Comment by fortunatomaldonado** [comment]
Resent https://github.com/blzaugg/liferay-portal/pull/219

File: modules/apps/frontend-js/frontend-js-aui-web/src/main/resources/META-INF/resources/liferay/menu.js (hunk 1 above)
Comment (fortunatomaldonado) [reply modules/apps/frontend-js/frontend-js-aui-web/src/main/resources/META-INF/resources/liferay/menu.js]: Retested and seems to work fine without it.

**Pull request closed without merging**